CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## 0.24.0
 
-
+ * Added support to configure the thread pool size for `kubernetes-ops-pool`
```

Comment:
Please add empty line after the bullet point. Maybe you could also make it a bit more generic? Nobody will know what does this mean.

```suggestion
 * Added support to configure the worker thread pool size
 
```